### ISSUE TITLE
Tests for twisted.words should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7622]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7622|
|Type|enhancement|
|Created|2014-09-02 16:38:06Z|
|Branch|https://github.com/twisted/twisted/tree/words-tests-naming-7622|

This ticket is a followup to <[#7032](https://github.com/twisted/twisted/issues/7032)>.

Attachments:

* [bug-7622.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/25d/25d1d5479c7da2c2a500a6a52ce909819e7c040b/99bf124a6c8944feaed42ba10563a88ae219b8aa.patch) (21024 bytes) - added by eeshangarg on 2014-09-02 16:47:53Z - 
* [bug-7622_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/25d/25d1d5479c7da2c2a500a6a52ce909819e7c040b/e90e642c271eaf46a011f0895fdcd8542fe2fa30.patch) (20431 bytes) - added by eeshangarg on 2014-09-20 15:33:08Z - 
* [bug-7622_3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/25d/25d1d5479c7da2c2a500a6a52ce909819e7c040b/b428b995861e3393b16c3b5dba169d231253b449.patch) (20539 bytes) - added by eeshangarg on 2014-11-19 12:03:08Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7622 7622
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_words_tests_naming_7622 branches/words-tests-naming-7622
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__words words
keywords__easy easy
time__1409675886250325 1409675886250325
changetime__1417019430330792 1417019430330792
version__None None
owner__ 
cc__eeshangarg cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_2"></a>|@eeshangarg commented|
|-|-|

Hi! I just added a patch that changes all of the tests in twisted.words to use the new style of naming test classes as discussed in this ticket <[#7032](https://github.com/twisted/twisted/issues/7032)>.

Thanks!
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_5"></a>|@eeshangarg set owner to @glyph|
|-|-|

Hi!

I changed all tests for twisted.words to use the FooTests naming scheme for test classes. The patch I would like to get reviewed is called bug-7622_2.patch​.

Thanks & regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_6"></a>|@glyph removed owner|
|-|-|

Un-assigning all review tickets assigned to me.  Unless there's a specific reason _I_ need to review something, let's leave them unassigned :-).

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_7"></a>|@eeshangarg commented|
|-|-|

Hi!

I just submitted a new patch for this called bugfix_7622_3.patch, which includes a .misc news file along with the contents of the previous patch. :)

Thanks & regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

(In [43493]) Branching to 'words-tests-naming-7622'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun commented|
|-|-|

(In [43494]) Apply bug-7622_3.patch 

refs [#7622](https://github.com/twisted/twisted/issues/7622)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun commented|
|-|-|

(In [43495]) Fix incorrect name in reference.

refs [#7622](https://github.com/twisted/twisted/issues/7622)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun commented|
|-|-|

Thanks.  I fixed one more name-related issues I noticed in the code.  The twistedchecker builder reports a ton of new errors but I think they're all actually just old errors with new names.  The rest of buildbot is green enough.  I'll merge this now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_12"></a>|@exarkun set status to `closed`|
|-|-|

(In [43498]) Merge words-tests-naming-7622 

Author: eeshangarg
Reviewer: exarkun
Fixes: [#7622](https://github.com/twisted/twisted/issues/7622)

Rename all of the test case classes in Twisted Words to conform to the
recently documented test case class naming convention.
